apps/api/src/server/repositories/document.repository.js

```diff
@@ -27,3 +27,25 @@ export const update = (documentGuid, documentDetails) => {
 		data: documentDetails
 	});
 };
+
+/**
+ *
+ * @param {string} documentGUID
+ * @returns {import('@prisma/client').PrismaPromise<import('@pins/api').Schema.Document | null>}
+ */
+export const getByDocumentGUID = (documentGUID) => {
+	return databaseConnector.document.findUnique({
+		where: { guid: documentGUID }
+	});
+};
+
+/**
+ * @param {{guid: string, status: import('xstate').StateValue }} documentStatusUpdate
```

Comment:
I am not sure it makes sense to sent a `StateValue` as the new status. A `string` makes more sense surely?

Reply:
agree - I changed it to string